R/utilities.R

```diff
@@ -4406,9 +4355,13 @@ classify_fl <- function(
 
 }
 
-#' Classify DLBCLs according to genetic subgroups of Chapuy et al.
+#' Classify DLBCLs according to genetic subgroups.
 #'
-#' Use the feature weights from NMF model to assemble the binary matrix and classify DLBCL tumors based on C0-C5 system of Chapuy et al
+#' Using the user-provided or GAMBLR-retrieved data, this function will assemble the matrix according to the approach of
+#' Chapuy et al (2018) or Lacy et al (2020) classifiers. Since neither of this classifiers is publicly released, we have implemented a solution
+#' that closely (> 92% accuracy) recapitulates each of these systems. For the classifier of Chapuy et al, the constructed matrix will be
+#' used to calculate class probability using the feature weights from bundled NMF model. For the Lacy et al classifier, the matrix will be
```

Comment:
Clarify what a bundled NMF model is. I thought NMF was clustering and your models were all RF

Reply:
Will be updated in the next commit